### ISSUE TITLE
 spacegraphcats/doc/installing-spacegraphcats.md is out of date

### ISSUE THREAD

**Issue by taylorreiter** [spacegraphcats/spacegraphcats#382]
After installing spacegraphcats, running 

```
python -m spacegraphcats dory-test search
```
produces the error
```
Usage: __main__.py [OPTIONS] COMMAND [ARGS]...
Try '__main__.py --help' for help.

Error: No such command 'dory-test'.
```

**Comment by taylorreiter** [comment]
```
(sgc_latest) tereiter@bm3:~/github/spacegraphcats$ python -m spacegraphcats dory-test search
Usage: __main__.py [OPTIONS] COMMAND [ARGS]...
Try '__main__.py --help' for help.

Error: No such command 'dory-test'.
(sgc_latest) tereiter@bm3:~/github/spacegraphcats$ python -m spacegraphcats run dory-test search

This is spacegraphcats 2.0b12.dev7+gc495b99.

Migrating .snakemake folder to new format...
Migration complete
Building DAG of jobs...
Nothing to be done.
Complete log: /home/tereiter/github/spacegraphcats/.snakemake/log/2021-05-21T085308.955391.snakemake.log

-------- DONE --------

catlas output directory: dory_k21_r1
search output directory: dory_k21_r1_search_oh0
```

**Comment by taylorreiter** [comment]
```
(sgc_latest) tereiter@bm3:~/github/spacegraphcats$ make twofoo.fq.gz
curl -L 'https://osf.io/7az9p/?action=download' > shew-reads.abundtrim.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   459  100   459    0     0     94      0  0:00:04  0:00:04 --:--:--    96
100 26.0M  100 26.0M    0     0  4356k      0  0:00:06  0:00:06 --:--:-- 26.0M
curl -o akker-reads.abundtrim.gz -L https://osf.io/dk7nb/download
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   459  100   459    0     0   1357      0 --:--:-- --:--:-- --:--:--  1357
100 27.9M  100 27.9M    0     0  16.0M      0  0:00:01  0:00:01 --:--:-- 35.7M
gunzip -c shew-reads.abundtrim.gz akker-reads.abundtrim.gz | gzip -9c > twofoo.fq.gz
(sgc_latest) tereiter@bm3:~/github/spacegraphcats$ python -m spacegraphcats twofoo search
Usage: __main__.py [OPTIONS] COMMAND [ARGS]...
Try '__main__.py --help' for help.

Error: No such command 'twofoo'.
```

```
(sgc_latest) tereiter@bm3:~/github/spacegraphcats$ python -m spacegraphcats run twofoo search

This is spacegraphcats 2.0b12.dev7+gc495b99.

Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Job counts:
        count   jobs
        1       bcalm_catlas_prepare_input
        1       bcalm_catlas_sort
        1       bcalm_cdbg
        1       bcalm_cdbg_inpfiles
        1       build_catlas
        1       make_contigs_kmer_index
        1       search
        7
```

**Comment by ctb** [comment]
fixed by #392, I think
